lua/debugprint/init.lua

```diff
@@ -213,8 +213,7 @@ M.debugprint_regular = function(opts)
     return "g@l"
 end
 
----@param opts DebugprintFunctionOptions
----@return nil
```

Comment:
I believe this was a mistake, as the function seems to return `string | nil`. The type can be automatically inferred to fix this.

Reply:
This was a mistake, although I prefer to use explicit types, so I would add `---@return string|nil` here.

Reply:
Ok, I just changed it to `string | nil`.

I was surprised the type checking actually works here - I have seen many cases where it doesn't 🙂 
So I agree, maybe better to keep it here.